src/patternfly/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,188 @@
+@import "../../patternfly-utilities";
+
+.pf-c-tabs-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+// Tabs
+.pf-c-tabs {
+  // Primary Tab Items
+  --pf-c-tabs__item--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs__item--current--Color: var(--pf-global--active-color--100);
+  --pf-c-tabs__item--hover--BoxShadowColor: #8b8d8f;
+  --pf-c-tabs__item--hover--BoxShadow: inset 0 pf-size-prem(2) 0 0 var(--pf-c-tabs__item--hover--BoxShadowColor);
+  --pf-c-tabs__item--current--BoxShadow: inset 0 pf-size-prem(2) 0 0 var(--pf-c-tabs__item--current--Color);
+  --pf-c-tabs__item--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs__item--BorderColor: #ededed;
+
+  // Secondary Tab Items
+  --pf-c-tabs__item--hover--Color: var(--pf-global--Color--dark-200);
+
+  // Tab Buttons
+  --pf-c-tabs__button--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // Scroll Buttons
+  --pf-c-tabs__scroll-button--Width: var(--pf-global--spacer--xl);
+  --pf-c-tabs__scroll-button--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs__scroll-button--BorderColor: #ededed;
+
+  position: relative;
+  display: inline-flex;
+
+  // Overlap secondary tab border
+  & ~ .pf-c-tabs.pf-m-tabs-secondary {
+    margin-top: -1px;
+  }
+
+  // Filled style
+  &.pf-m-fill {
+    display: flex;
+    width: 100%;
```

Comment:
I think you can just use `display: flex;` or `width: 100%;` here? I would probably use `display: flex;`

Reply:
You're right, I've removed the `width: 100%;`.